packages/styles/variables.css

```
@@ -127,6 +127,9 @@
   /* radio card */
   --radio-card-width: 255px;
   --radio-card-height: 229px;
+
+  /* Font family for the project*/
+  --base-font-family: 'Roboto', Helvetica, Arial, sans-serif;
```

Minor nit:
```suggestion
  /* Fonts */
  --base-font-family: 'Roboto', Helvetica, Arial, sans-serif;
```

...and move it near the top since it's a shared global thing.